### PR TITLE
REFS Maintenance Team - Fixing spec dummy contributions yml missing.

### PR DESCRIPTION
**Pull request by ZASMan**
### Master Redmine ticket
* (Required!)

### Child Redmine ticket(s)
* ()
* ()

### Local build result

```
bundle exec rake parallel:spec[4]
bundle exec cucumber
```

### Latest rebase/merge tag
*

---

### Data ticket(s) Followup
* (redmine links here - optional)

### Related Pull Requests
* (github links here - optional)

---

#### Peer Review
* (For code review)

#### Functional Testing
* (For testing locally)

#### Deployment
* (for release manager and/or build manager)

### CODE REVIEW

File: components/benefit_markets/spec/dummy/system/config/templates/features/aca_shop_market/contributions.yml [hunk 1]
```diff
@@ -226,7 +226,7 @@ registry:
         is_enabled: true
         settings:
           - key: :contribution_model_key
-            item: :fifty_percent_sponsor_fixed_percent_contribution_model
+            item: :zero_percent_sponsor_fixed_percent_contribution_model
```
Comment (harshared) [inline components/benefit_markets/spec/dummy/system/config/templates/features/aca_shop_market/contributions.yml:229]: why changed this?

**Comment by harshared** [comment]
Please merge latest master.

File: components/benefit_markets/spec/dummy/system/config/templates/features/aca_shop_market/contributions.yml (hunk 1 above)
Comment (ZASMan) [reply components/benefit_markets/spec/dummy/system/config/templates/features/aca_shop_market/contributions.yml:229]: Accident, I was just copying and pasting.

File: components/sponsored_benefits/spec/dummy/system/config/templates/features/aca_shop_market/contributions.yml [hunk 2]
```diff
@@ -226,7 +226,7 @@ registry:
         is_enabled: true
         settings:
           - key: :contribution_model_key
-            item: :fifty_percent_sponsor_fixed_percent_contribution_model
+            item: :zero_percent_sponsor_fixed_percent_contribution_model
```
Comment (harshared) [inline components/sponsored_benefits/spec/dummy/system/config/templates/features/aca_shop_market/contributions.yml:229]: here too.

Comment (harshaellanki) [reply components/sponsored_benefits/spec/dummy/system/config/templates/features/aca_shop_market/contributions.yml:229]: synced dummy to main app

**Pull request merged**